src/steps/os/linux.rs

```diff
@@ -157,13 +157,14 @@ fn upgrade_arch_linux(ctx: &ExecutionContext) -> Result<()> {
     debug!("Running Arch update with path: {:?}", path);
 
     if let Some(yay) = which("yay").or_else(|| which("paru")) {
-        run_type
```

Comment:
I rather have some thing like 
```rust 
let yay = which("yay")
if let Some(yay) = yay {
// Show news
}

if let Some(yay) = yay.or_else(|| which ("paru")) {
// Upgrade
}
````

Reply:
Thanks for the feedback, I implemented it in 4b11094